spec/features/edit_update_account_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+require 'rails_helper'
+
+feature 'user edits/deletes account', %(
+  As an authenticated user
+  I want to update my information
+  So that I can keep my profile up to date
+) do
+
+  scenario 'updating account information' do
+    user = FactoryBot.create(:user)
+    visit new_user_session_path
```

Comment:
Leave a space after creating variables, use `let` syntax.

Reply:
@jdmacmurtrie  I thought in ruby we don't use `let`?